app/services/gitlab_repo_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rest-client'
+require 'json'
+require 'uri'
```

Comment:
Are these necessary?

Reply:
umm, no 🤔 
removed them
I was following the basic usage example from [rest-client readme](https://github.com/rest-client/rest-client#usage-raw-url). 

Is rails doing something here?

Reply:
Unless you add `require: false` next to a gem, it is usually available globally without needing a `require` line

Reply:
I see. Got it